autoload/suda.vim

```diff
@@ -9,13 +9,24 @@ function! suda#system(cmd, ...) abort
   if v:shell_error == 0
     return result
   endif
-  try
-    call inputsave()
-    redraw | let password = inputsecret(g:suda#prompt)
-  finally
-    call inputrestore()
-  endtry
-  let cmd = printf('%s -p '''' -S %s', g:suda#executable, a:cmd)
+  " Let's try running a command non-interactively. If it works, we have a sudo
+  " timestamp that has not timed out yet. In this case there is no need to ask
+  " for a password.
+  " This only works if the timestamp_type is set to 'global' in the sudo
+  " configuation file. It does not work with 'ppid', 'kernel' or 'tty'.
```

Comment:
Please explain this behavior and limitation in `doc/suda.txt` as well.

Reply:
Done. Please let me know if you are ok with an extra commit like that or you want me to squash it.